### DISCUSSION TITLE
what is web interface for?

### DISCUSSION THREAD

**Discussion by kavemang** [beetbox/beets#4170] [Q&A] [answered]
I have beets installed on docker and I'm able to use it just fine from command line, but the web interface seems to be blank, it has a query text entry and a large blank white space and I have no idea what its meant to do

**Comment by martinkirch** [comment, marked as answer]
It's a Web player :

 - type a query - artist name, title, album name,... [anything that `beet ls` would understand](https://beets.readthedocs.io/en/stable/reference/query.html)
 - press `Enter`
 - matching results will appear in the left bar
 - double click to play

By the way, I've started a discussion on this interface, feel free to give some feedback in #4336
